src/main/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientConfiguration.java

```diff
@@ -42,6 +44,12 @@
         required = true)
     String url();
 
+    @AttributeDefinition(
+        name = "Default HTTP method",
+        description = "The default HTTP method used to send GraphQL requests.",
+        required = true)
+    HttpMethod httpMethod() default HttpMethod.POST;
```

Comment:
We can use the `options` of `@AttributeDefinition` here to provide all possible values (get, post). Felix console will render it as a dropdown selection

Reply:
This is done automatically when it's an `enum`.

Reply:
Cool did not know that. Nice.